build.cake

```diff
@@ -114,12 +113,7 @@ Task("__Pack")
         CreateDirectory(odNugetPackDir);
         CopyFileToDirectory(Path.Combine(assetDir, nuspecFile), odNugetPackDir);
 
-		CopyFileToDirectory(solutionDir + "Server" + bin452 + "Octopus.Server.Extensibility.Authentication.DirectoryServices.dll", odNugetPackDir);
-        // files = new [] {
-            // solutionDir + "Server" + bin452 + "Octopus.Node.Extensibility.Authentication.DirectoryServices.dll",
-            // solutionDir + "Server" + bin452 + "Octopus.Server.Extensibility.Authentication.DirectoryServices.dll"
-        // };
-        // Zip("./", Path.Combine(odNugetPackDir, "DirectoryServices.zip"), files);
+		CopyFileToDirectory(solutionDir + "Server" + netstd + "Octopus.Server.Extensibility.Authentication.DirectoryServices.dll", odNugetPackDir);
```

Comment:
Nitpick: Indentation

Reply:
No problems. Whitespace was all over the place in that file, have fixed.